Sources/SymbolKit/UnifiedSymbolGraph/GraphCollector.swift

```diff
@@ -26,11 +26,38 @@ public class GraphCollector {
     var graphSources: [String: [GraphKind]] = [:]
 
     var extensionGraphs: [URL: SymbolGraph] = [:]
+    
+    private let strategy: MergeStrategy
 
-    public init() {
+    /// Initialize a new collector for merging ``SymbolGraph``s into ``UnifiedSymbolGraph``s.
+    ///
+    /// - Parameter strategy: Optionally specifiy how graphs are to be merged.
+    public init(strategy: MergeStrategy = MergeStrategy()) {
         self.unifiedGraphs = [:]
         self.graphSources = [:]
         self.extensionGraphs = [:]
+        self.strategy = strategy
+    }
+}
+
+extension GraphCollector {
+    /// Describes the strategy for how to merge symbol graph files.
+    public struct MergeStrategy {
```

Comment:
I initially passed the strategy directly to `GraphCollector/mergeSymbolGraph(_:at:forceLoading:)`, which already has a couple of parameters. So, in anticipation of more strategy options that might be introduced in the future, I thought this design would be more tidy.

However, since the strategy is now passed to GraphCollector's initializer, which didn't have any parameters before, it looks a bit overly abstracted now. 👍